tests/test_enable_disable.py

```diff
@@ -0,0 +1,46 @@
+from random import choice
+
+from sphinxlint import main
+
+BULLET = "- "
+
+
+def test_default(capsys):
+    main(["sphinxlint", "--list"])
+    out, _err = capsys.readouterr()
+    assert out.count(BULLET) > 10
+
+
+def test_disable_all(capsys):
+    main(["sphinxlint", "--disable", "all", "--list"])
+    out, _err = capsys.readouterr()
+    assert out.count(BULLET) == 0
```

Comment:
Does this print a message like `No checkers selected.` (or `N checkers selected: ...` when some are selected)?

Reply:
No. `assert not out` would pass. It may not be human friendly though...

I like your idea of `N checkers selected:`, ... implementing it... done:

```
$ python sphinxlint.py --enable all --list
22 checkers selected:
- block-bad-dedent: Check for dedent not being enough in code blocks.
- carriage-returns: Check for carriage return in lines.
- default-role: Search for default roles (but they are allowed in many projects).
- directives-with-three-dots: Search for directives with three dots instead of two.
- directives-without-colon: Search for directive wrongly typed as comments, like: .. versionchanged 3.6.
- horizontal-tabs: Check for horizontal tabs in lines.
- hyperlinks-missing-space-before-less-than: Search for hyperlinks missing a space, like: `Link text<https://example.com>`.
- hyperlinks-missing-underscore-after-closing-backtick: Search for hyperlinks missing underscore after their closing backtick.
- inline-literal-glued-on-the-right: Search for inline literals glued to plural, like ``items``s.
- leaked-markup: Check HTML files for leaked reST markup; this only works if
- line-too-long: Check for line length; this checker is not run by default.
- missing-space-after-roles: Search for roles immediately followed by a character, like :exc:`Exception`s.
- newline-at-end-of-file: Check if the last line ends with a newline, like any other lines.
- python-syntax: Search invalid syntax in Python examples.
- roles-missing-colon-before-backtick: Search for roles missing a colon, like: :issue`123`.
- roles-missing-last-backtick: Search for roles missing their closing backticks, like :fct:`foo.
- roles-missing-leading-space: Search for spaces missing before roles, like the:fct:`sum`.
- roles-preceded-with-backtick: Search for roles preceded by a backtick, like `:fct:`sum`.
- roles-with-double-backtick: Search for roles with double backticks like: :fct:``sum``.
- roles-without-backticks: Search roles without backticks like :func:pdb.main.
- trailing-whitespaces: Check for trailing whitespaces at end of lines.
- triple-backticks: Check for triple backticks, like ```Point``` (but it's a valid syntax).

$ python sphinxlint.py --disable all --list
No checkers selected.
```